static/app/views/eventsV2/table/queryField.tsx

```diff
@@ -605,9 +630,7 @@ class BufferedInput extends React.Component<InputProps, InputState> {
   };
 
   handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
-    if (this.isValid) {
```

Comment:
why was this removed?

Reply:
since the regex is `auto|[0-9]*(\.[0-9]*)?` typing letters like `a` or `au` are considered invalid so the field won't have its state set. The other option was to make the regex something like `auto|aut|au|a|[0-9]*(\.[0-9]*)?` so that the intermediate states are valid, but that's kinda gross

Reply:
This breaks the existing validation on `number` and `integer` inputs 🤔 an actual example would be that it allows you to enter `asdf` as the second argument to `percentile` now, but because of the way `handleBlur` is implemented it resets the value correctly.

I'm a personally little torn on this. If someone else has some thoughts, feel free to weigh in.

Reply:
yeah, basically `handleBlur` handles the validation now

Reply:
I'm also not 100% good on this, open to better suggestions

Reply:
but on the other hand field validation does happen as soon as you click out of the field which doesn't seem like the worst thing. Especially because all the inputs into these fields are relatively short like 3-4 characters so you're not losing much by not getting real time validation as you type. anything longer than 3-4 characters (like columns) is in a dropdown